src/public/crud/js/ajax.js

```diff
@@ -0,0 +1,89 @@
+$(document).on('hidden.bs.modal', '.modal', function () {
+    var modalData = $(this).data('bs.modal');
+
+    if (modalData && modalData.options.remote) {
+        $(this).removeData('bs.modal');
+        $(this).find(".modal-content").empty();
+    }
+});
+
+crudForm.submit(function(event) {
+    event.preventDefault();
+
+    $(this).ajaxSubmit({
+        dataType: 'json',
+        success: function(response, statusText, xhr, form) {
+            if (xhr.status == 200 && !$.isEmptyObject(response))
+            {
+                new PNotify({
+                    text: response.message,
+                    type: 'success'
+                });
+
+                $('#crudTable').dataTable().api().ajax.reload();
+
+                switch (saveActionField.val()) {
+                    case 'save_and_new':
+                    case 'save_and_edit':
+                        $("#modal-ajax-crud .modal-content").load(response.redirect_url, function() {
+                            $("#modal-ajax-crud").modal("show"); 
+                        });
+                        break;
+                    case 'save_and_back':
+                    default:
+                        $("#modal-ajax-crud").modal("hide");
+                        break;
+                }
+            }
+        },
+        error: function(xhr, statusText, error) {
```

Comment:
This doesn't seem to fire?

Reply:
https://github.com/Laravel-Backpack/CRUD/pull/1149#issuecomment-351241020